custom_components/flashforge_adventurer_3/protocol.py

```diff
@@ -35,48 +43,54 @@ async def send_msg(reader: StreamReader, writer: StreamWriter, payload: str):
     logger.debug(f'Response from the printer: {result}')
     return result.decode()
 
-
-async def collect_data(ip: str, port: int) -> Tuple[PrinterStatus, Optional[str], Optional[str]]:
+async def collect_data(ip: str, port: int, model: str) -> Tuple[PrinterStatus, Optional[str], Optional[str]]:
```

Comment:
Let's use strong typing for `model` rather than passing in any arbitrary string. Either a string literal or an enum can work.